### PR TITLE
[callback url support] 

### PR DESCRIPTION
**Pull request by mayankamencherla**
Added processing url to magento

### CODE REVIEW

File: js/razorpay/razorpay-utils.js [hunk 1]
```diff
@@ -33,7 +33,8 @@ function RazorpayUtils(options) {
                 name: this.options.customer_name,
                 contact: this.options.customer_phone,
                 email: this.options.customer_email
-            }
+            },
+            callback_url: this.options.callback_url,
```
Comment (captn3m0) [inline js/razorpay/razorpay-utils.js:37]: This needs to be handled on the backend

**Review by captn3m0 — COMMENTED** [review]
error conditions are not handled in callback flow

File: app/code/community/Razorpay/Payments/Helper/Data.php [hunk 2]
```diff
@@ -141,21 +141,31 @@ public function createOrder($order)
...
-        $responseArray['customer_phone']    = $bA->getTelephone() ?: '';
-        $responseArray['order_id']          = $orderId;
-        $responseArray['base_amount']       = $amount;
-        $responseArray['base_currency']     = $base_currency;
-        $responseArray['customer_email']    = $order->getData('customer_email') ?: '';
-        $responseArray['quote_currency']    = $quote_currency;
-        $responseArray['quote_amount']      = $quote_amount;
+        $responseArray = array(
+            // order id has to be stored and fetched later from the db or session
+            'customer_name'     => $bA->getFirstname() . " " . $bA->getLastname(),
```
Comment (ccrims0n) [inline app/code/community/Razorpay/Payments/Helper/Data.php]: please us single quotes.

**Review by ccrims0n — APPROVED** [review]
lgtm

**Pull request merged**